ui/app/store/actions.js

```diff
@@ -2798,3 +2825,79 @@ function setSeedPhraseBackedUp (seedPhraseBackupState) {
     })
   }
 }
+
+function hideSeedPhraseBackupAfterOnboarding () {
```

Comment:
It looks like this was added accidentally.

Reply:
Oh wait... nevermind. This is needed - it was just missing. That's concerning.

Reply:
In fact, it is not needed. The case where it appears in the code is never called. I've removed the addition of this via a rebase, and will remove the case where it is called in a separate PR.

Reply:
It looks like this is still here - did it get re-added?

Reply:
It looks like this should be here - it's just broken on develop right now :/